### ISSUE TITLE
Error when invoking lookup_order with more than one OID: "OID incorrecto"

### ISSUE THREAD

**Issue by fdbatista** [bitsoex/bitso-php#6]
Hello.
I am having problems with passing arrays to the `bitso` methods.
I am still facing #5 and now this one. Here is the thing:

I have two active orders on Bitso for testing whose OIDs are `Ai2LrQIJXUSP7Ykq` and `2hYAYgElws0b0RLO`.
When i try to execute the `lookup_order` method like this:
```php
$response = $this->bitsoAPI->lookup_order(['Ai2LrQIJXUSP7Ykq', '2hYAYgElws0b0RLO']);
``` 
I get the error: `OID incorrecto (no existe o no pertenece al usuario)`

Which is not true. Both IDs exist and belong to the proper user.
Actually, when I make two separate calls like this:
```php
$response = $this->bitsoAPI->lookup_order(['Ai2LrQIJXUSP7Ykq']);
$response = $this->bitsoAPI->lookup_order(['2hYAYgElws0b0RLO']);
```
I get the correct response, an object with two attributes: `success` and `payload`.

So, what is going on here?
Why invoking methods with arrays of objects is not working for `lookup_order`? The same happens with `cancel_order`, I have a pending PR here to propose a solution :)

Invoking one request per each OID is not a good option because, as you can imagine, it would take much more time, and time is a highly valuable resource when dealing with volatile cryptoassets.

**Comment by fdbatista** [comment]
@julyanserra Hi, mate. Looks like there is another bug in this method.

Line 391 in `bitso.php` class contains this sentence:
`$parameters = implode('', $ids);`

As you can see, all OIDs passed to the method are being concatenated in a single string wihout delimiter.
I made a test changing the `glue` parameter to `,` and it works fine. It returns the correct payload with both orders' status.

That makes two PR already for this repository :)

How come nobody found this before?

**Comment by fdbatista** [comment]
I've created a PR in #7. Please take a look at it. Cheers.

**Comment by domix** [comment]
closed by #7
